maquetado/scss/styles.scss

```diff
@@ -39,8 +41,8 @@
 
 .redirect-arrow {
   font-size: 20px;
-  margin-left: 0 0 60px 100px;
```

Comment:
If you are deleting this then you should add `align-self: left` to move it to the left. This is how it looks now

![image](https://user-images.githubusercontent.com/36488701/59800083-b382b600-92b3-11e9-8ed1-3994d221c1c7.png)

With `align-self` is enough to fix it since you are using display grid on your body. You should also add `align-self: center`